tests/test_config_parsing/test_res_config.py

```diff
@@ -292,6 +293,34 @@ def test_that_a_config_warning_is_given_when_eclbase_and_jobname_is_given():
         ErtConfig.from_file(test_config_file_name)
 
 
+@pytest.mark.usefixtures("use_tmpdir")
+def test_parsing_forward_model_with_double_dash_is_possible():
+    """This is a regression test, making sure that we can put double dashes in strings.
+    The use case is that a file name is utilized that contains two consecutive hyphens,
+    which by the ert config parser used to be interpreted as a comment. In the new
+    parser this is allowed"""
+
+    test_config_file_name = "test.ert"
+    test_config_contents = dedent(
+        """
+        NUM_REALIZATIONS  1
+        JOBNAME job_%d--hei
+        FORWARD_MODEL COPY_FILE(<FROM>=foo,<TO>=something/hello--there.txt)
+        """
+    )
+    with open(test_config_file_name, "w", encoding="utf-8") as fh:
+        fh.write(test_config_contents)
+
+    res_config = ErtConfig.from_file(
+        user_config_file=test_config_file_name, use_new_parser=True
```

Comment:
Very minor but, omit use of `user_config_file=` for arg to be consistent with other use in file?

Reply:
Done :)